src/Database/Traits/SortableRelation.php

```diff
@@ -0,0 +1,140 @@
+<?php namespace Winter\Storm\Database\Traits;
+
+use Exception;
+
+/**
+ * SortableRelation model trait
+ *
+ * Usage:
+ *
+ * In the model class definition add:
+ *
+ *   use \Winter\Storm\Database\Traits\SortableRelation;
+ *
+ *   public $sortableRelations = ['relation_name' => 'sort_order_column'];
+ *
+ * To set orders:
+ *
+ *   $model->setSortableRelationOrder($relationName, $recordIds, $recordOrders);
+ *
+ */
+trait SortableRelation
```

Comment:
This should probably be `HasSortableRelations`

Reply:
done